pyrk/inp/sim_info.py

```diff
@@ -32,7 +32,7 @@ def __init__(self,
         self.rho_ext = self.init_rho_ext(rho_ext)
         self.feedback = feedback
         self.ne = self.init_ne()
-        self.th = th_system.THSystem(kappa=kappa, components=components)
+        self.th = th_system.THSystemSphFVM(kappa=kappa, components=components)
```

Comment:
How should we go about this problem, let user specify which THSystem the simulation should use? Then we need to give some help documentation somewhere, because the class name is not very obvious